src/components/widgets/sales-leads-funnel/sales-leads-funnel.tmpl.html

```diff
@@ -4,7 +4,7 @@
     <h4>Widget settings</h4>
 
     <div setting-organizations parent-widget="widget" class="part" deferred="::orgDeferred" />
-    <div setting-params-picker options="statusOptions" param="status_selection" parent-widget="widget" class="part" deferred="::paramsPickerDeferred" />
+    <div setting-params-picker options="statusOptions" param="leads_per_status" has-reach="hasReach" parent-widget="widget" class="part" deferred="::paramsPickerDeferred" />
```

Comment:
Hi,
No: the response you will receive from the API won't necessarily match the parameters you send.
For example, the parameter status_selection you send to the API refers to the list of status that you want to take into account. The parameter leads_per_status refers to a list of leads, ordered per status.
So it's not going to be the same.

Please revert back to status_selection